views/Home/Home.jsx

```diff
@@ -108,6 +111,19 @@ export default class HomeView extends React.Component {
     this.props.navigation.navigate('Feed', { channel, loggedInUser: user });
   }
 
+  onLongPressChannel = (channelId, channelName) => {
+    const { channels, user } = this.state;
+    var channel;
+
+    if (['all', 'subs', 'myPosts'].includes(channelId)) channel = { _id: channelId, name: channelName };
```

Comment:
Not directly related to this PR but just making a general note:
It might make sense to just append these channels to `state.channels` here instead of in `ChannelList.jsx` so we don't have to do this custom logic

Reply:
Actually maybe that's a smell that this logic should be in ChannelList, since Home probably shouldn't need to known about these channels. I don't see any reason why this method can't be defined in `ChannelList.jsx`.

No strong opinion though

Reply:
I'm not really familiar with this, so I'm gonna leave this for another PR, if that's okay